.github/workflows/ci.yml

```diff
@@ -34,14 +34,11 @@ jobs:
           - ubuntu-latest
           - macos-latest
           - windows-latest
-        arch:
-          - x64
```

Comment:
Currently, CI on macos-latest does not use the Apple silicon but the x64 binaries. This change fixes it, now CI uses the native binaries on each runner.